src/analytics.ts

```diff
@@ -171,15 +174,9 @@ function getEventProperties(eventName: string, target: HTMLElement) {
 /**
  * It is possible for browsers to block the Sentry script from being downloaded, so capture messages safely.
  */
-function safelyCaptureMessage(message: string, level: SeverityLevel = null, context: SentryContext = null) {
+function safelyCaptureMessage(message: string, level: SeverityLevel = null) {
     if (typeof Sentry !== 'undefined') {
-        Sentry.withScope(function (scope) {
```

Comment:
Nothing in the callback was running, so events weren't being sent. We only used the scope for the A/B testing, which is currently inactive, so I've just disabled the context for now and added a TODO in the Google Analytics block.

Reply:
That's reasonable!